pmda/test/test_rmsf.py

```diff
@@ -35,3 +35,10 @@ def test_RMSF_n_jobs(u, n_cores):
     assert_almost_equal(MDA_vals.mean, PMDA_vals.mean)
     assert_almost_equal(MDA_vals.sumsquares, PMDA_vals.sumsquares)
     assert_almost_equal(MDA_vals.rmsf, PMDA_vals.rmsf)
+
+
+@pytest.mark.parametrize('rmsf_array', [pytest.param(np.array([-1, -1, -1]),
+                                        marks=pytest.mark.xfail)])
```

Comment:
Are you testing that this method raises an error? Then you can do the following
```python
def test_negative_RMSF_raises_ValueError():
   with pytest.raises(ValueError):
      pmda.rmsf.RMSF._negative_rmsd(np.array([0, -1, -2]))
```

- Test directly that an exception is raised.
- Directly use the static method.